rollup.config.js

```diff
@@ -1,30 +1,33 @@
-import {terser} from 'rollup-plugin-terser';
+import json from '@rollup/plugin-json';
+import { terser } from 'rollup-plugin-terser';
 import typescript from '@rollup/plugin-typescript';
 
 export default {
   input: 'src/embed/init.ts',
   output: [
     {
       format: 'iife',
-      file: 'build/dlo.js'
+      file: 'build/dlo.js',
     },
     {
       file: 'build/dlo.min.js',
       format: 'cjs',
       name: 'version',
       plugins: [terser({
-        enclose: true
-      })]
-    }
+        enclose: true,
+      })],
+    },
   ],
   plugins: [
+    json(),
     typescript({
-      target: "es5",
-      module: "es2015",
+      target: 'es5',
+      module: 'es2015',
       declaration: false,
       declarationMap: false,
       sourceMap: false,
-      resolveJsonModule: false
-    })
-  ]
-};
\ No newline at end of file
+      resolveJsonModule: true,
+      moduleResolution: 'node',
```

Comment:
Changing the module resolution strategy was necessary to import `package.json` via the `@rollup/plugin-json` plugin. Versioning DLO is already a little burdensome:
- `npm version`
- Add `CHANGELOG` entry
- Update version in `README`
- After PR merges, tag merged version and push new version tag

I wanted to lean on the version from `package.json` to avoid adding another step (e.g. update version in `telemetry.ts`).